controllers/kubevirtcluster_controller_test.go

```diff
@@ -22,14 +22,14 @@ import (
 )
 
 var (
-	mockCtrl *gomock.Controller
-	clusterName         string
-	kubevirtClusterName string
```

Comment:
no logic is changing in this file.  If this isn't related to the PR, revert this please.